sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2170,7 +2171,22 @@ object GpuOverrides {
       (a, conf, p, r) => new UnaryExprMeta[MakeDecimal](a, conf, p, r) {
         override def convertToGpu(child: Expression): GpuExpression =
           GpuMakeDecimal(child, a.precision, a.scale, a.nullOnOverflow)
-      })
+      }),
+    expr[CollectList](
+      "Collect a list of elements, now only supported by windowing.",
+      /* It should be 'fullAgg' eventually but now only support windowing,
+         so 'aggNotGroupByOrReduction' */
+      ExprChecks.aggNotGroupByOrReduction(
+        TypeSig.ARRAY.nested(TypeSig.commonCudfTypes + TypeSig.DECIMAL + TypeSig.STRUCT),
+        TypeSig.ARRAY.nested(TypeSig.all),
+        Seq(ParamCheck("input",
+          (TypeSig.commonCudfTypes + TypeSig.DECIMAL).nested() + TypeSig.STRUCT,
+          TypeSig.all))),
+      (c, conf, p, r) => new ExprMeta[CollectList](c, conf, p, r) {
+        override def convertToGpu(): GpuExpression = GpuCollectList(
+          childExprs.head.convertToGpu(), c.mutableAggBufferOffset, c.inputAggBufferOffset)
+      }).disabledByDefault("for now the GPU collects null values to a list, but Spark does not." +
```

Comment:
@sameerz I just wanted to check with you that this is an acceptable short term solution.  That we disable the operator and the user will have to explicitly set `spark.rapids.sql.expression.CollectList=true` to use it.

Reply:
Yes this works for now.